src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,48 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the response json with the
+ * aggregated buildbot data.
+ */
+public class BuildInfo {
+  private final String commitHash;
+  private final String timeStamp;
```

Comment:
Is there a reason why this field is a string, and not some java-native way to represent a timestamp? I'd prefer one of those here.

Reply:
The time stamp is a sequence of numbers, I decided to use String instead of long or int because this number could be huge and even be out of range.

Reply:
It's surprising to me that the time stamp is a sequence of numbers. How does that sequence encode time?
Note that Java provides classes like [Date](https://docs.oracle.com/javase/7/docs/api/java/util/Date.html) and there's also other some modern libs like [joda-time](https://www.joda.org/joda-time/) that provide a Java-native timestamp class. I'd expect Spring's JSON mapping capabilities to include support for mapping JSON timestamp fields in various formats to such native Java objects, e.g., https://stackoverflow.com/questions/10286204/the-right-json-date-format

Reply:
Thanks! I changed it to Data format, it will be very convenient especially if we decide to display it in the future.